internal/gitlab/gitlab.go

```diff
@@ -604,6 +604,25 @@ func IssueClose(pid interface{}, id int) error {
 	return nil
 }
 
+// IssueDuplicate closes an issue as duplicate of another
+func IssueDuplicate(pid interface{}, id int, dupId string) error {
+	// Not exposed in API, go through quick action
+	body := "/duplicate " + dupId
```

Comment:
I wasn't able to find about that in the API docs too.. but seems a good enough solution to use the 'quick action' mechanism :-)

Reply:
I also looked at the raw json output for an issue that was closed as duplicate, and the issue it duplicated. There's really nothing there :shrug: